README.md

```diff
@@ -1,9 +1,15 @@
-# Node Maintenance Operator
-
-The node-maintenance-operator is an operator generated from the [operator-sdk](https://github.com/operator-framework/operator-sdk).
-The purpose of this operator is to watch for new or deleted custom resources called NodeMaintenance which indicate that a node in the cluster should either:
-  - NodeMaintenance CR created: move node into maintenance, cordon the node - set it as unschedulable and evict the pods (which can be evicted) from that node.
-  - NodeMaintenance CR deleted: remove pod from maintenance and uncordon the node - set it as schedulable.
+# Deprecation Notice
+
+**March 17, 2022**
+  
+NMO's **KuebVirt** is deprecated in favor of [NMO's Medik8s](https://github.com/medik8s/node-maintenance-operator), 
```

Comment:
**KubeVirt's** Node Maintenance Operator (NMO) is deprecated in favor of [**Medik8s'** NMO]... ?

Reply:
It does make more sense :D

Reply:
An additional s is missing?
> is deprecated in favor of [Medik8s' NMO]... ?

 is deprecated in favor of [Medik8s's NMO]... ?

Reply:
indeed, TIL